cmd/gcContent/gcContent.go

```diff
@@ -0,0 +1,96 @@
+// Command Group: "FASTA and Multi-FASTA Tools"
+
+// Calculates the gc content of fasta sequences for all regions specified in a bed file
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/dna"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/fileio"
+	"log"
+)
+
+func gcContent(bedFile string, faFile string, outFile string, multiFaMode bool, species string) {
```

Comment:
I would consider a more descriptive name, that lets people know that the program is specifically looking at GC content within user specified bed regions.

Reply:
Changed name to bedGcContent